pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -20,16 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/go-openapi/spec"
```

Comment:
This has the same issue as the one @smarterclayton had pointed out in last PR.
We are adding the openapi dependency everywhere.
Anyone using this authenticator now pulls in the dependency.